packages/frontend/src/components/UserSettingsModal/ProjectManagementPanel/index.tsx

```diff
@@ -17,33 +26,87 @@ import {
 const ProjectListItem: FC<{
     isCurrentProject: boolean;
     project: OrganizationProject;
-}> = ({ isCurrentProject, project: { projectUuid, name } }) => (
-    <ProjectListItemWrapper elevation={0}>
-        <ItemContent>
-            <ProjectInfo>
-                <ProjectName className={Classes.TEXT_OVERFLOW_ELLIPSIS}>
-                    {name}
-                </ProjectName>
-                {isCurrentProject && (
-                    <ProjectTag minimal>Current Project</ProjectTag>
-                )}
-            </ProjectInfo>
-            <ButtonGroup>
-                <LinkButton
-                    icon="cog"
-                    outlined
-                    text="Settings"
-                    href={`/projects/${projectUuid}/settings`}
-                />
-            </ButtonGroup>
-        </ItemContent>
-    </ProjectListItemWrapper>
-);
+}> = ({ isCurrentProject, project: { projectUuid, name } }) => {
+    const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
+    const { mutate, isLoading: isDeleting } = useDeleteProjectMutation();
+    return (
+        <ProjectListItemWrapper elevation={0}>
+            <ItemContent>
+                <ProjectInfo>
+                    <ProjectName className={Classes.TEXT_OVERFLOW_ELLIPSIS}>
+                        {name}
+                    </ProjectName>
+                    {isCurrentProject && (
+                        <ProjectTag minimal>Current Project</ProjectTag>
+                    )}
+                </ProjectInfo>
+                <ButtonGroup>
+                    <LinkButton
+                        icon="cog"
+                        outlined
+                        text="Settings"
+                        href={`/projects/${projectUuid}/settings`}
+                    />
+                    <Button
+                        icon="trash"
```

Comment:
In the design and in the user management panel we use the `delete` icon. Maybe we should use it here to keep it consistent.